docs/slides/content/estimation.md

```diff
@@ -31,58 +31,60 @@ TL500 <!-- {.element: class="title-color"} -->
 
 
 
 #### _Why use it?_
 
 * Facilitates conversation
-* Helps align groups of people to reach consensus on their understanding regarding the size of complexity and/or value of an item
-* Avoids the emotion, buffering and other negative side effects of absolute estimation
+* Helps align groups to reach consensus on their understanding regarding the size of, complexity, and/or value of an item
+* Avoids the emotion, buffering, and other negative side effects of absolute estimation
 
 
 
 #### _How do we do it?_
-_Things to consider before starting:_ <!--{.element: style="font-size: smaller; font-weight: 100;"} -->
-* First we need to have done the product backlog refinement and the sprint planning
-* After we picked the user stories for the next sprint and put them into the Relative Estimation board
+_Things to consider before starting_ <!--{.element: style="font-size: smaller; font-weight: 100;"} -->
+* First, the product backlog refinement and the sprint planning must be done.
+* Then, you pick up the user stories for the next sprint and put them into the Relative Estimation board.
 
 
 
 #### _How do we do it?_
-1. We draw a ladder and agree on the points we are going to use. A common practice is to use the Fibonacci series or T-Shirt sizes
+1. Draw a ladder and agree on the points to use. A common practice is to use the Fibonacci series or T-Shirt sizes.
 
 ![fibonacci-series](images/estimation/fibonacci-series.png)<!-- {.element: class="" style="border:none; box-shadow:none; max-width:40%;" } -->
 
-2. Then we start the sizing for the stories: the first step is to identify the 1-pointer and the 13-pointer
-3. From there we then compare the other user stories to the previous items:
-    - Maybe the second user story is a bit harder, so let's call that a 2-pointer
-    - Then the next user story we need to ask ourselves how much harder or easier it is compared to the others
+2. Then, begin sizing for the stories. The first step is to identify the 1-pointer and the 13-pointer.
+3. Next, compare the other user stories to the previous items:
+    - Perhaps the second user story is a bit harder, so let's call that a 2-pointer
+    - Consider the story that follows. We must ask ourselves how much harder or easier it is compared to the others.
 
 
 
 #### _How do we do it?_
 _An example_ <!--{.element: style="font-size: smaller; font-weight: 100;"} -->
-* 'Okay we have story A, is this bigger than the 3 points we have here?'
-* Assuming yes - 'Okay is it bigger than the 8 pointer? Or is it roughly the same size'
-* Assuming no - 'Okay so we'll say it is a 5 pointer then'
+* 'Is Story A bigger than the 3 points we have here?'
+* Assuming yes, ask "Is it bigger than the 8 pointer? Or is it roughly the same size?"
+* Assuming no, "Okay, then let's say it is a 5 pointer."
+<!--If it's not bigger than story A, then why do we increase the points from 3 to 5? This is unclear.-->
```

Comment:
its about relative sizing .. we are merely asking if this is bigger than an 8 point story we may have
we know its bigger than the 3 point story .. but how much bigger ?